soem/ethercatmain.c

```diff
@@ -370,6 +370,10 @@ uint8 ecx_siigetbyte(ecx_contextt *context, uint16 slave, uint16 address)
          /* 4 byte response */
          else
          {
+#ifdef EC_BIG_ENDIAN
```

Comment:
I get you point but I'd like to make it more clear what's going on. How about making it more explicit?

   uint64 edat64;
   uint32 edat32;
...
         edat64 = ecx_readeepromFP (context, configadr, eadr, EC_TIMEOUTEEP);
         /* 8 byte response */
         if (context->slavelist[slave].eep_8byte)
         {
            put_unaligned64(edat64, &(context->esibuf[eadr << 1]));
            cnt = 8;
         }
         /* 4 byte response */
         else
         {
            edat32 = (uint32)edat64;
            put_unaligned32(edat32, &(context->esibuf[eadr << 1]));
            cnt = 4;
         }

Reply:
You are right, will do